api/v1alpha1/podiochaos_types.go

```diff
@@ -124,6 +131,35 @@ type AttrOverrideSpec struct {
 	Rdev *uint32 `json:"rdev,omitempty"`
 }
 
+// MistakeSpec represents one type of mistake
+type MistakeSpec struct {
+	// Filling determines what is filled in the miskate data.
+	// +optional
+	// +kubebuilder:validation:Enum=zero;random
+	Filling FillingType `json:"filling,omitempty"`
+
+	// There will be [1, MaxOccurrences] segments of wrong data. If MaxOccurrences is 0, no mistake will be injected
+	// +optional
+	// +kubebuilder:validation:Minimum=1
+	MaxOccurrences int64 `json:"maxOccurrences,omitempty"`
```

Comment:
If we set `Mininum=1`, how about remove the comments about `MaxOccurrences = 0`

Reply:
fixed